File: exploring_docker/cpp/hello.cpp
#include <iostream>
#include <ctime>

int main() {
    std::time_t t = std::time(nullptr);
    std::tm* now = std::localtime(&t);
    std::cout << "Hello ASL!" << std::endl;
    std::cout << (now->tm_year + 1900) << "-" << (now->tm_mon + 1) << "-" << now->tm_mday << std::endl;
    return 0;
}

File: exploring_docker/rust/hello.rs
use std::time::{SystemTime, UNIX_EPOCH};
use chrono::prelude::*;

fn main() {
    // Get the current local date and time
    let local: DateTime<Local> = Local::now();

    // Print the greeting
    println!("Hello ASL!");

    // Print the current date in the format "YYYY-MM-DD"
    println!("{}", local.format("%Y-%m-%d"));
}
